source/patterns/@aws-solutions-constructs/core/lib/waf-helper.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { Construct } from "@aws-cdk/core";
+import * as api from '@aws-cdk/aws-apigateway';
+import * as waf from "@aws-cdk/aws-wafv2";
+import { DefaultWafRules, DefaultWafwebacl } from "./waf-defaults";
+
+export interface BuildWebaclProps {
+     /**
+      * The existing API Gateway instance that will be protected with the WAF web ACL.
+      */
+     readonly existingApiGatewayInterface: api.IRestApi;
```

Comment:
Looking at the code below, it appears there is no longer a need for this member of this interface.